src/uct/tcp/tcp_cm.c

```diff
@@ -305,14 +304,13 @@ uct_tcp_cm_simult_conn_accept_remote_conn(uct_tcp_ep_t *accept_ep,
     status = uct_tcp_ep_move_ctx_cap(accept_ep, connect_ep,
                                      UCT_TCP_EP_CTX_TYPE_RX);
     if (status != UCS_OK) {
-        return status;
+        return 0;
     }
 
-    /* 3. Destroy the EP allocated during accepting connection
+    /* 3. Caller has to destroy the EP allocated during accepting connection
```

Comment:
what is meant by `caller`?

Reply:
I meant the function that calls this procedure
Looks like it's the bad name.
Changed to:
```
3. The EP allocated during accepting connection has to be destroyed upon return from this function
```